[16.0][ADD] crm referred by categorized (#591)
OCA/crm: crm_referred/readme/USAGE.md

To use this module, you need to:

Opportunity behavior:

> 1. Go to **CRM > Configuration > CRM Referred**.

Reviewer: ```suggestion
1. Go to **CRM > Configuration > CRM Referrers**.
```

> 1. Create news records.

Reviewer: ```suggestion
1. Create a new record.
```

1. Go to **CRM > Sales > My Pipeline**.
1. Create new opportunity or use a created opportunity.

> 1. Fill field **Referenced by Categorized**. Field is in *Extra Information* page in *Marketing* section.

Reviewer: ```suggestion
1. Fill field **Categorized referrer**. Field is in *Extra Information* page in *Marketing* section.
```

Lead behavior (only if leads are activated):

1. Go to **CRM > Configuration > CRM Referred**.

> 1. Create news records.

Reviewer: ```suggestion
1. Create a new record.
```

1. Go to **CRM > Lead**.

> 1. Create new lead or use a created lead.

Reviewer: ```suggestion
1. Create a new lead or use a created lead.
```

> 1. Fill field **Referenced by Categorized**. Field is in *Extra Info* page in *Marketing* section.

Reviewer: ```suggestion
1. Fill field *Categorized referrer*. Field is in *Extra Info* page, in *Marketing* section.
```

1. Click on **CONVERT TO OPPORTUNITY**.
1. In wizard choose action *Convert to opportunity* and choose *Create new customer* in Customer section.

> 1. New partner has same **Referenced by Categorized**

Reviewer: ```suggestion
1. The new partner has the same *Categorized referrer*
```
